tests/testthat/test-sparsevctrs.R

```diff
@@ -94,6 +110,11 @@ test_that("sparse tibble can be passed to `predict()", {
 
   hotel_data <- sparse_hotel_rates()
   hotel_data <- sparsevctrs::coerce_to_sparse_tibble(hotel_data)
+    
+  # materialize outcome
```

Comment:
```suggestion
  withr::local_options("sparsevctrs.verbose_materialize" = NULL)
  # materialize outcome
```

This test currently depends on the option not having been set.